static/bidder-info/ownadx.yaml

```diff
@@ -1,4 +1,5 @@
-endpoint: "https://pbs.prebid-ownadx.com/bidder/bid/{{.AccountID}}/{{.ZoneID}}?token={{.SourceId}}"
+#endpoint: "https://pbs.prebid-ownadx.com/bidder/bid/{{.AccountID}}/{{.ZoneID}}?token={{.SourceId}}"
+endpoint: "https://pbs.prebid-ownadx.com/bidder/bid/{{.SeatID}}/{{.SspID}}?token={{.TokenID}}"  # as per suggestion endpoint change
```

Comment:
Please remove the comments. New macro choice looks fine.

Reply:
Remove Comment from Yaml file